### DISCUSSION TITLE
Export in PDF Through JDF Job Option

### DISCUSSION THREAD

**Discussion by Cicorione** [speedata/publisher#615] [Q&A]
Hi Patrick,

I am planning to do a demonstration about Speedata at my new job however all my supervisors are product managers without a design background, it means they do not understand very well the whole design process and have difficult to go beyond what they were instructed of.

One the weird things that are happening here is that all the PDF workflow, even the one that goes digital, is totally intertwined with Cockpit Prinect by Heidelberg. Each PDF must processed with a specific JDF option, we have JDF options for our offset printers plus the one for the digital printers.

Wherever test I am going to run with Speedata I need to be able to export with the JDF option without reprocessing the PDF through acrobat.

What do you do when your clients needs to export directly with a JDF profile?

Thanks... 🙏

**Comment by pgundlach** [comment]
I have to admit that I have never heard of a JDF profile. (Actually this is not strictly true, but I never dealt with such a profile). So I cannot share any experience with that.

I hope I can help you, but I don't know how.

**Reply by Cicorione** [reply]
The options needed on indesign are very trivial but the actual documentation has 922 pages, this is crazy... 😱

https://www.cip4.org/files/cip4/documents/JDF%20Specification%201.8%20www.pdf

**Comment by pgundlach** [comment]
I see that there is a JDF editor at https://github.com/cip4/JDFEditor, but I have no idea how to use it. I can open it and add elements and such, but I don't know what I should do.

Is JDF necessary to start the speedata Publisher or is it necessary to move the resulting PDF to some location? 

(me <-- totally clueless)

**Reply by Cicorione** [reply]
From an Indesign point of view it is just a matter to save some settings, like bleeds, color conversion, etc... The personalization is saved as **joboption**.. 

I guess it would be more or less the same with Publisher if I could set up the same settings:

https://www.lifewire.com/joboptions-file-2621848
https://helpx.adobe.com/indesign/using/pdf-options.html

Maybe I get confused and JDF and joboption **aren't the same thing**. 🤔
